minecraft_servers/craftportal/manifest.json

```diff
@@ -3,11 +3,11 @@
   "nice_name": "Craftportal",
   "direct_ip": "craftportal.net",
   "supported_languages": [
-    "de"
+    "de", "en"
   ],
   "social": {
-      "discord":"https://discord.gg/7F6MbnUSEp",
-      "teamspeak":"ts.craftportal.net",
+      "web": "https://www.craftportal.net",

```

Comment:
If website is in maintenance mode, there's no real meaning behind having it in here
If the website will be in maintenance for longer, remove